treeherder/webapp/api/artifact.py

```diff
@@ -58,6 +59,9 @@ def create(self, request, project):
         job_guids = [x['job_guid'] for x in request.DATA]
         with JobsModel(project) as jobsModel, ArtifactsModel(project) as artifacts_model:
 
+            # create an accompanying ``Bug suggestions`` artifact for any eligible artifacts
```

Comment:
Yeah, good call.  OK, changed that.  I went back and forth on this, but this is more explicit to the user.